### DISCUSSION TITLE
How to make visual-repeat `.` works in `V-LINE` mode?

### DISCUSSION THREAD

**Discussion by bchen** [LazyVim/LazyVim#2245] [Q&A] [answered]
In V-LINE mode, `.` can't work after I type `>` on lines.
I'd like to know which default plugin or setting disabled it?

**Comment by dpetka2001** [comment]
You have to press `.` from normal mode. It works for me.

**Reply by bchen** [reply, marked as answer]
Thanks @dpetka2001, that is true, but visual-repeat works on vim and nvim by default.

After checking the source code, it should caused by https://github.com/LazyVim/LazyVim/blob/main/lua/lazyvim/config/keymaps.lua#L73-L75
```
-- better indenting
map("v", "<", "<gv")
map("v", ">", ">gv")
```

nvm, I can override it if needed.
